moon/controller_round3.py

```diff
@@ -546,19 +554,24 @@ def run(self):
 #            self.go_straight(-2, timeout=timedelta(seconds=20))
 #            self.turn(math.radians(45), timeout=timedelta(seconds=20))
 #            self.set_cam_angle(CAMERA_ANGLE_HOMEBASE)
-#            self.follow_object(['basemarker'])
-#            self.current_driver = 'basemarker'
+
 
 
             if SKIP_CUBESAT_SUCCESS:
                 # skip cubesat
                 self.cubesat_success = True
                 self.follow_object(['homebase'])
             else:
-                # regular launch
+               # regular launch
```

Comment:
why is this? the older version seems correctly aligned

Reply:
I did not do this explicitly, it was probably a side-effect of some bulk commenting out of things